Polyhedron/demo/Polyhedron/Plugins/Classification/CMakeLists.txt

```diff
@@ -6,7 +6,12 @@ if(EIGEN3_FOUND)
   set(classification_compile_definitions)
   
   find_package(Boost OPTIONAL_COMPONENTS serialization iostreams)
-  if (Boost_SERIALIZATION_FOUND AND Boost_IOSTREAMS_FOUND)
+  find_package( ZLIB QUIET )
+  if( NOT ZLIB_FOUND AND WIN32 )
+    find_package( Boost OPTIONAL_COMPONENTS zlib)
+  endif()
+
+  if (Boost_SERIALIZATION_FOUND AND Boost_IOSTREAMS_FOUND AND (ZLIB_FOUND OR Boost_ZLIB_FOUND))
```

Comment:
You forgot to add Zlib to `${classification_linked_libraries}`.

Reply:
You are right, I fixed it.